### PR TITLE
[Android] "AudioSource" option to choose a specific audio source (noise filtering)

### PR DESCRIPTION
**Pull request by Ketn**
Simple and small snippet to get a recording without background noise in android.

### CODE REVIEW

**Comment by jsierles** [comment]
Thanks, this looks useful! What do you think about allowing setting this constant directly, so other values can be used?

**Comment by Ketn** [comment]
That was my first intention, however I'm pretty much a noob when it comes to working with native modules. I don't know how to refer to android specific constants etc. If you could direct me, I'll go ahead and implement that.

**Comment by Ketn** [comment]
@jsierles I've added the option to set the constant directly. Please check.

File: index.js [hunk 1]
```diff
@@ -89,6 +91,9 @@ if (Platform.OS === 'ios') {
     DocumentDirectoryPath: AudioRecorderManager.NSDocumentDirectoryPath,
     LibraryDirectoryPath: AudioRecorderManager.NSLibraryDirectoryPath,
   };
+  AudioSource = {
```
Comment (jsierles) [inline index.js]: Why is this added here as an empty dictionary?

Comment (Ketn) [reply index.js]: I forgot to remove it after I added initialization at the top. My bad.

**Comment by jsierles** [comment]
The author needs to fix the last thing mentioned here.

**Comment by ketnatnoon** [comment]
I've fixed the required changes. @jsierles , this PR is ready to merge, please do so.

**Comment by Ketn** [comment]
I've fixed the remaining issue. @jsierles , please merge this PR, thanks!
